miden-lib/asm/note_scripts/SWAP.masm

```diff
@@ -63,7 +68,14 @@ begin
     drop drop drop movdn.4
     # => [ASSET, tag, RECIPIENT]
 
+    push.PUBLIC_NOTE movdn.5
+    # => [ASSET, tag, note_type, RECIPIENT]
+
     # create a note using inputs
     call.wallet::send_asset dropw dropw
+    # => [ptr, ZERO, ZERO, 0]
+
+    # clean stack
+    dropw dropw drop drop
```

Comment:
This was a bug, the test started failing because with the addition of the `note_type` we started to overflow the stack